src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const tabIdActive = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Comment:
tabIdActive is not so good name for this const as you find whole tab not only id